cmd/kops/create_cluster.go

```diff
@@ -300,7 +299,7 @@ func NewCmdCreateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd.RegisterFlagCompletionFunc("subnets", completeSubnetID(options))
 	cmd.Flags().StringSliceVar(&options.UtilitySubnetIDs, "utility-subnets", options.UtilitySubnetIDs, "Shared utility subnets to use")
 	cmd.RegisterFlagCompletionFunc("utility-subnets", completeSubnetID(options))
-	cmd.Flags().StringVar(&options.NetworkCIDR, "network-cidr", options.NetworkCIDR, "Network CIDR to use")
+	cmd.Flags().StringSliceVar(&options.NetworkCIDRs, "network-cidr", options.NetworkCIDRs, "Network CIDR(s) to use")
```

Comment:
Do we need to make this change here as well - https://github.com/kubernetes/kops/blob/master/tests/e2e/kubetest2-kops/deployer/up.go#L112?

Reply:
I don't see any mention of `network-cidr` there.

Reply:
IIUC, kubetest2 uses that interface which I linked earlier and we would need to provide this flag there as well.

Reply:
I think we can pass them via the "passthrough" CreateArgs: https://github.com/kubernetes/kops/blob/9e0db3db74fbe5e435bbf7d1af805ab07f34bf6e/tests/e2e/kubetest2-kops/deployer/up.go#L126-L135